pkgs/development/python-modules/playwright/default.nix

```diff
@@ -0,0 +1,235 @@
+{ lib
+, stdenv
+, autoPatchelfHook
+, callPackage
+, fetchFromGitHub
+, fetchurl
+, makeWrapper
+, runCommand
+, unzip
+
+# Node is required for the driver
+, nodejs
+
+# Python dependencies
+, python
+, buildPythonPackage
+, pythonOlder
+, autobahn
+, greenlet
+, objgraph
+, pillow
+, pixelmatch
+, pyee
+, pyopenssl
+, requests
+, service-identity
+, websockets
+
+# For building the browser bundle
+, cacert
+, chromium
+, ffmpeg
+, makeFontsConf
+}:
+
+let
+  inherit (stdenv.hostPlatform) system;
+
+  version = "1.22.0";
+
+  driver = let
+    suffix = {
+      x86_64-linux = "linux";
+      aarch64-linux = "linux-arm64";
+      x86_64-darwin = "mac";
+      aarch64-darwin = "mac-arm64";
+    }.${system};
+    filename = "playwright-${version}-${suffix}.zip";
+  in stdenv.mkDerivation {
+    inherit version;
+    pname = "playwright-driver";
+
+    src = fetchurl {
+      url = "https://playwright.azureedge.net/builds/driver/${filename}";
+      sha256 = {
+        x86_64-linux = "sha256-OcdPLxXFg/WlEPxeCLoQyYmqM/dQN3BFSVeFkTxzYeQ=";
+        aarch64-linux = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
+        x86_64-darwin = "sha256-roXTpFyPIhFLsupLW9Fvc4iEkzdG9b8wOOzoSVytxV0=";
+        aarch64-darwin = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
```

Comment:
The update script doesn't fill the hashes in if you run it?

Reply:
Oh yeah, I should probably complete the script an add the remaining hashes… I'll try to get around to doing it in the next few days